lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -245,6 +262,41 @@ class AwsCompileCloudFrontEvents {
         );
       }
 
+      // Check if all behaviors got unique pathPatterns
+      if (behaviors.length !== _.uniqBy(behaviors, 'PathPattern').length) {
+        throw new Error('Found more than one behavior with the same PathPattern');
```

Comment:
Aren't those validation errors picked already by `CloudFormation.validateTemplate` SDK command, which we run internally?

Reply:
Unfortunatelly not